src/components/TopBar/TopBar.tsx

```diff
@@ -0,0 +1,49 @@
+import Select from 'react-select';
+
+import { useWorkspace } from '@/hooks/useWorkspace';
+import './TopBar.scss';
+
+const TopBar = () => {
+  const { availableWorkspaces, activeWorkspace, setActiveWorkspace } = useWorkspace();
+
+  return (
+    <div className="top-bar">
+      <div className="workspace-selection">
+        <label className="workspace-label" htmlFor="workspace-select">
+          Change Workspace
+        </label>
+        <Select
+          className="workspace-select"
+          defaultValue={{ value: activeWorkspace, label: activeWorkspace }}
+          id="workspace-select"
+          options={availableWorkspaces.map((workspace) => ({ value: workspace, label: workspace }))}
+          theme={(theme) => ({
+            ...theme,
+            borderRadius: 0,
+            colors: {
+              ...theme.colors,
+              primary: '#2a3559',
+              primary25: '#2a355930',
+            },
+          })}
+          onChange={(selectedOption) => setActiveWorkspace(selectedOption?.value)}
+        />
+      </div>
+
+      <div>
+        <p className="heading">Your Workspace</p>
+        <p className="date">
+          {activeWorkspace} -{' '}
+          {new Date().toLocaleDateString('en-US', {
```

Comment:
Is there a reason we are using the US Date? I normally have battles on this as I prefer iso 8601 e.g. 2024-09-18 as opposed to the constant battles I've had over US/UK dates. The world seems pretty split between the 3 formats, but only iso 8601 I feel is unambiguous. Again, not asking for change, more a question really.